bin/dart2_fix.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:io';
 import 'package:dart2_fix/src/dart_fix.dart';
 import 'package:dart2_fix/src/model.dart';
 
-main(List<String> args) async {
+void main(List<String> args) async {
```

Comment:
This should be `Future` :) (because its an async method)

Reply:
Oh I see, thank you :)